platform/ios/platform/ios/src/MGLMapView.mm

```diff
@@ -598,19 +753,44 @@ - (void)commonInit
     
     self.anchorRotateOrZoomGesturesToCenterCoordinate = NO;
     
+#if TARGET_IPHONE_SIMULATOR
+    bool isM1Simulator = processIsTranslated() > 0;
```

Comment:
I suppose it's not just M1 but an M1 using Rosetta (translated)?

Reply:
Grab doesn't have the arm64 version Map, now I Updated the macro test only effect to x86_64 & x86, I think it's the condition that would happen this issue (An m1 simulator & x86_64 architecture code).

Reply:
@nvanfleet Do you have any other suggestion to setup the guard?

Reply:
@nvanfleet , can this be resolved in your opinion?

Reply:
These guards are much better, but how do we know that this is an m1 simulator, and not just a developer using an x86_64 mac with a x86_64 simualtor?